### PR TITLE
translated italian strings

### LINKED ISSUE #87: Update Italian translation
Strings that are missing Italian translation

```xml
<string name="settings_step_vibrate_item">Step change vibration</string>
<string name="settings_backup_item">Backup/Restore</string><string name="settings_timer_item">Timer settings</string>
<string name="prepopulate_aero_description">Recipe created by: James Hoffmann Source: https://youtu.be/j6VlT_jUVPc Grind size: For light roast – finer end of medium. The darker you go the more you may prefer to increase the dose and coarsen the grind Temperature: For light roast – brought to a boil and used immediately. For dark roasts – 85–90℃</string>
<string name="prepopulate_aero_step_press">Press the piston gently</string>
<string name="prepopulate_aero_step_piston">Place the piston 1cm into the brewer</string>
<string name="prepopulate_aero_step_piston">Place the piston 1cm into the brewer</string>
<string name="prepopulate_aero_name">AeroPress</string>
<string name="settings_addDefault">Add default recipes</string>
<string name="settings_restore">Restore</string>
<string name="settings_backup">Backup</string>
<string name="step_add_duration_explainer">Step duration determines how long an action will take (in seconds). If you want to create a step that will hold whole recipe till you are read, just leave it empty.</string>
<string name="recipe_details_noTime">Press start to continue</string>
<string name="button_show">Show</string>
<string name="recipe_edit_description_button">Add description</string>
<string name="app_version">App version</string>
<string name="button_continue_editing">Continue editing</string>
<string name="recipe_clone_suffix">%1$s - Copy</string>
<string name="recipe_clone_text">This will create a duplicate of this recipe</string>
<string name="recipe_clone_title">Clone this recipe?</string>
<string name="button_discard">Discard</string>
<string name="recipe_unsaved_text">Do you want to discard them?</string>
<string name="recipe_unsaved_title">You have unsaved changes</string>
```

After you translated them, add them to `app/src/main/res/values-it/strings.xml` and send a PR

Please do not use online translation services for this!

### PR DESCRIPTION
**Pull request by l1i1s7a**
# Description

Fixed #87 (Update Italian translation)

## Type of change
- translated missing italian strings and added them at the bottom of `app/src/main/res/values-it/strings.xml`

### CODE REVIEW

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -66,4 +66,28 @@ Occhio a non bloccare la fuoriuscita dell'aria dal coffee brewer"</string>
     <string name="tereszkiewicz_credits_subtitle">Guarda il suo Dribbble</string>
     <string name="settings_step_sound_item">Emetti un suono tra i passaggi</string>
     <string name="recipe_create_title">Aggiungi ricetta</string>
+
+
```
Comment (rozPierog) [inline app/src/main/res/values-it/strings.xml:70]: Could you remove empty new lines from here? You can do that even from here \/
```suggestion
```

**Review by rozPierog — CHANGES_REQUESTED** [review]
Thanks! 
Just one minor nitpick before merging

File: app/src/main/res/values-it/strings.xml (hunk 1 above)
Comment (l1i1s7a) [reply app/src/main/res/values-it/strings.xml]: removed the new lines!

**Review by rozPierog — APPROVED** [review]
Grazie!

**Pull request merged**